apps/ca.c

```diff
@@ -214,17 +213,21 @@ const OPTIONS ca_options[] = {
 
     OPT_SECTION("Signing"),
     {"md", OPT_MD, 's', "md to use; one of md2, md5, sha or sha1"},
-    {"keyfile", OPT_KEYFILE, 's', "Private key"},
-    {"keyform", OPT_KEYFORM, 'f', "Private key file format (PEM or ENGINE)"},
+    {"keyfile", OPT_KEYFILE, 's', "The CA private key"},
+    {"keyform", OPT_KEYFORM, 'f',
+     "Private key file format (PEM|DER|P12|ENGINE); default PEM"},
     {"passin", OPT_PASSIN, 's', "Input file pass phrase source"},
-    {"key", OPT_KEY, 's', "Key to decode the private key if it is encrypted"},
+    {"key", OPT_KEY, 's', "Key to decode the private key or cert files if encrypted. Better use -passin"},
     {"cert", OPT_CERT, '<', "The CA cert"},
-    {"certform", OPT_CERTFORM, 'F',
-     "certificate input format (DER or PEM); default PEM"},
+    {"certform", OPT_CERTFORM, 'f',
+     "Certifiate file format (PEM|DER|P12|HTTP); default PEM"},
```

Comment:
Typo : _Certifiate_

Reply:
Ah, you caught my most notorious typo :sweat_smile: